documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -280,6 +284,69 @@ Previously this property was only checked by the `rpi-eeprom-update` script. How
 
 Default: `0`
 
+[[HTTP_HOST]]
+==== HTTP_HOST
+
+If an HTTP download is initiated, `boot.img` and `boot.sig` are downloaded from this server.
+
+Invalid host names will be ignored. They should only contain lower case alphanumeric characters and `-` or `.`.
+If `HTTP_HOST` is set then HTTPS is disabled and plain HTTP used instead.
+You can specify an IP address to avoid the need for a DNS lookup.
+Don`t include the HTTP scheme or any forward slashes in the hostname.
+
+Default: `fw-download-alias1.raspberrypi.com`
+
+[[HTTP_PORT]]
+==== HTTP_PORT
+
+You can change the port used for HTTP downloads. This is ignored if `HTTP_HOST` is not set.
+
+By default network install and HTTP boot mode use port `443` because HTTPS is enabled when using the default host `fw-download-alias1.raspberrypi.com`. If `HTTP_HOST` is changed then HTTPS is not supported and HTTP will be used instead, so port `80` becomes the default.
```

Comment:
As I think I suggested before, it might be worth clarifying that if the user has set `HTTP_HOST`, then plain HTTP will still be used even if they change `HTTP_PORT` to 443 ?

Reply:
reworded all this